src/fmu/tools/_common.py

```diff
@@ -50,3 +52,17 @@ def force_stop(string):
         print()
 
         sys.exit(string)
+
+
+def preserve_cwd(func):
```

Comment:
I think the `monkeypatch.chdir` would do the same as this function

Reply:
Thanks. Will add as an issue